src/Nervos/Entry.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright © 2017-2022 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "Entry.h"
+
+#include "Address.h"
+#include "Planner.h"
+#include "Signer.h"
+
+using namespace TW;
+using namespace TW::Nervos;
+using namespace std;
+
+bool Entry::validateAddress(TWCoinType coin, const string& address, byte p2pkh, byte p2sh,
+                            const char* hrp) const {
+    return Address::isValid(address, p2pkh, p2sh, hrp);
```

Comment:
no need to pass p2sh/p2pkh
```suggestion
    return Address::isValid(address, hrp);
```

Reply:
Done